### PR TITLE
CoE Starter kit Added

### PR DESCRIPTION
**Pull request by snova-kwasia**

### CODE REVIEW

File: fine_tuning_embeddings/config.yaml [hunk 1]
```diff
@@ -32,13 +32,13 @@ asr:
...
     
-api: sambaverse
+api: sambastudio
 
 llm: 
     "temperature": 0.1
     "max_tokens_to_generate": 1024
     "sambaverse_model_name": "Mistral/Mistral-7B-Instruct-v0.2"
-    "smabaverse_select_expert": "Mistral-7B-Instruct-v0.2"
+    "samabaverse_select_expert": "Mistral-7B-Instruct-v0.2"
```
Comment (snova-amitk) [inline fine_tuning_embeddings/config.yaml:41]: spelling: "samabaverse"

File: CoE_jump_start/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+1. Clone the repository:
+
+  ```bash
+  git clone https://github.com/your-repo/calling-coe-models.git
+  ```
+
+2. Install the required dependencies::
+  ```bash
+  pip install -r requirements.txt
```
Comment (snova-amitk) [inline CoE_jump_start/README.md:32]: Do you want to have the python env creation also cpoied here before pip install.

Comment (snova-kwasia) [reply CoE_jump_start/README.md]: I've added the venv creation now. Thank you.

File: CoE_jump_start/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+The script and notebook support various models for calling CoE\, including\:
+
+\- Mistral\-7B\-Instruct\-v0\.2
+\- finance\-chat
+\- deepseek\-llm\-67b\-chat
+\- medicine\-chat
+\- law\-chat
+
+For a full list of supported models\, please refer to the \[Supported Models\]\(https\:\/\/link\-to\-supported\-models\) documentation\.
```
Comment (snova-jamesv) [inline CoE_jump_start/README.md:130]: https://docs.sambanova.ai/sambastudio/latest/samba-1.html#_samba_1_expert_models

**Pull request merged**